webapp/app.py

```diff
@@ -29,14 +32,20 @@
 
 @app.errorhandler(404)
 def page_not_found(e):
-    frontpage, nav_html = discourse.get_frontpage()
+    try:
+        frontpage, nav_html = discourse.parse_frontpage()
```

Comment:
if you don't use `frontpage` use `_` for multiple output functions with var that you don't use:

`_, nav_html = discourse.parse_frontpage()`

Reply:
@tbille I've seen that pattern, but I don't believe it's a particularly official one (unless you know of it being mentioned in a PEP?). I disagree with it - the `_` null variable isn't a part of the language - it's still an unused variable, so given we're going to be creating a variable either way, I think it's still more useful to name it what it actually is than not to tell anyone what it is.

Reply:
It's not a official pattern, but it's a common pattern (especially in functionnal programmation). I don't really mind, it's just for me it's clearer to have `_` because i tend to search where the var is used when I debug. With `_` I know immediatly that the var is not used, so it can be skipped

Reply:
Yeah, personally I think the semantic argument of properly explaining the variable carries more weight. Especially given there's no guarantee that the person reading the code understands what `_` is supposed to mean.

Reply:
this is a super common pattern in python and any functionnal language. But if you think it's confusing, I don't mind keeping it that way.